diff --git a/src/session_record.js b/src/session_record.js
index 7626a3927fadcdf7f66b3d0c9cf8c63d33da579b..0c53db66c89a50394e871fb8a8898577d6de9900 100644
--- a/src/session_record.js
+++ b/src/session_record.js
@@ -269,6 +269,6 @@ class SessionRecord {
         if (this.isClosed(session)) {
-            console.warn("Session already closed", session);
+            console.warn("Session already closed");
             return;
         }
-        console.info("Closing session:", session);
+        console.info("Closing session");
         session.indexInfo.closed = Date.now();
@@ -278,5 +278,5 @@ class SessionRecord {
         if (!this.isClosed(session)) {
             console.warn("Session already open");
         }
-        console.info("Opening session:", session);
+        console.info("Opening session");
         session.indexInfo.closed = -1;
@@ -300,3 +300,3 @@ class SessionRecord {
             if (oldestKey) {
-                console.info("Removing old closed session:", oldestSession);
+                console.info("Removing old closed session");
                 delete this.sessions[oldestKey];
